job/server/src/main/java/alluxio/worker/job/command/JobWorkerHealthReporter.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.worker.job.command;
+
+import oshi.SystemInfo;
+import oshi.hardware.CentralProcessor;
+
+/**
+ * The job worker health reporter.
+ */
+public class JobWorkerHealthReporter {
+
+  CentralProcessor mProcessor;
+
+  /**
+   * Default constructor.
+   */
+  public JobWorkerHealthReporter() {
+    mProcessor = new SystemInfo().getHardware().getProcessor();
+  }
+
+  /**
+   * @return the cpu load average of the worker
+   */
+  public double[] getCpuLoadAverage() {
+    return mProcessor.getSystemLoadAverage(3);
```

Comment:
I'm not sure if Alluxio ever ran on Windows machines, but it looks like this method will prevent it from running there

http://oshi.github.io/oshi/apidocs/oshi/hardware/CentralProcessor.html#getSystemLoadAverage-int-

Reply:
Also, when the 1/5/15 minute loads aren't available we get a negative number returned. Should we do any special handling for those cases?

Reply:
I know we only print them now - but in the future do we want to simply return the negative numbers, or do any kind of processing here so that we don't have different logic for processing negative numbers?

i.e. in the case of any negative numbers we could return an empty array, set the negative values to 0, etc..

Reply:
I don't think it's any different because if I use this for logic, I'll be comparing values to be greater than some value. I probably shouldn't change value of an api value until I need to.